loom/engine/bindings/loom/lmSQLite.cpp

```diff
@@ -32,13 +32,129 @@ using namespace LS;
 //SQLite Statement binding for Loomscript
 class Statement
 {
+private:
+ //  Connection c; //crashes the build? Not sure if its because it's only defined later in the file? Not sure about c++
```

Comment:
Would need to be Connection *c; 

In C/C++, you can't declare an object before it is defined, but you *can* declare a pointer to one.  Syntax may change a bit, so I'll check it out for you.